src/main/java/ledger/user_interface/ui_controllers/component/TransactionTableView.java

```diff
@@ -152,6 +153,7 @@ private void handleDeleteSelectedTransactionsFromTableView() {
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
+        this.getColumns().add(NoteColumn.noteColumn());
```

Comment:
Should just add column via the FXML. Should need no code changes in TrasactionTableView to implement feature.